api/envoy/api/v2/route/route.proto

```diff
@@ -455,6 +455,29 @@ message RouteAction {
     //   retry policy, a request that times out will not be retried as the total timeout budget
     //   would have been exhausted.
     google.protobuf.Duration per_try_timeout = 3 [(gogoproto.stdduration) = true];
+
+    message RetryPriority {
+      string name = 1 [(validate.rules).string.min_bytes = 1];
+      google.protobuf.Struct config = 2;
+    }
+
+    // Specifies an implementation of a RetryPriority which is used to determine the
```

Comment:
Since these aren't implemented, they should be tagged with `[#not-implemented-hide:]` (at the start of the comment for each field).